src/test/Operate.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import operate from '../logic/operate';
+import Header from '../components/nav';
+import Calculator from '../pages/Calculator';
+
+// test('Render Home and check quote text', async () => {
+//   render(<App />, { wrapper: BrowserRouter });
+//   const user = userEvent.setup();
+
+//   expect(screen.getByText(/calculator/i)).toBeInTheDocument();
+
+//   await user.click(screen.getByText(/quote/i));
+//   expect(screen.getByText(/- William Thurston/i)).toBeInTheDocument();
+// });
```

Comment:
- [ ] I suggest you remove commented code from your codebase for easier readability and for clean code.